scripts/run_docker.sh

```diff
@@ -5,14 +5,13 @@ export NEAR_HOME=/srv/near
 
 NEARD_FLAGS=${NEAR_HOME:+--home=$NEAR_HOME}
 
-if [[ -z {INIT} ]]
+if [[ -n "${INIT}" ]]
 then
     neard $NEARD_FLAGS init ${CHAIN_ID:+--chain-id=$CHAIN_ID} ${ACCOUNT_ID:+--account-id=$ACCOUNT_ID}
```

Comment:
While fixing the script, could you also quote the variables? Specifically `${CHAIN_ID:+--chain-id="$CHAIN_ID"}` and `${ACCOUNT_ID:+--account-id="$ACCOUNT_ID"}`.

Reply:
done